src/System.Private.Uri/tests/FunctionalTests/UriParserTest.cs

```diff
@@ -193,6 +193,7 @@ public static void GetComponents_Null()
         [Fact]
         public static void GetComponents_BadUriComponents()
         {
+            http = new Uri(full_http);
```

Comment:
I asked about it yesterday ... there was something about renaming the assembly exposed a bug with the uninitialized variable by changing test execution ordering.  It's a voodoo change but it checks out.